src/screens/CreateTokenConfirm.js

```diff
@@ -68,28 +68,46 @@ class CreateTokenConfirm extends React.Component {
    * @param {String} pinCode User PIN
    */
   executeCreate = (pin) => {
-    const address = this.props.wallet.getCurrentAddress({ markAsUsed: true });
-    const ret = this.props.wallet.createNewToken(
-      this.name, this.symbol, this.amount, address, { pinCode: pin }
-    );
+    const { address } = this.props.wallet.getCurrentAddress({ markAsUsed: true });
+    this.props.wallet.prepareCreateNewToken(
+      this.name, this.symbol, this.amount, { address, pinCode: pin }
+    ).then((tx) => {
+      let sendTransaction;
+      // TODO substitute for feature flag when rollout code is implemented
```

Comment:
Maybe create a constant that will be replaced later (e.g. `isWalletService`)?

Reply:
Created a redux variable that is being used now as feature flag. https://github.com/HathorNetwork/hathor-wallet-mobile/pull/111/commits/6f6c05de5d4714040ee97ffdd2dcddf37e9ac619

Reply:
Great!